collections/_general/accordion.html

```diff
@@ -37,7 +36,7 @@ <h5 class="has-text-weight-semibold">Simple accordion</h5>
     </div>
 </div>
 
-<h5 class="has-text-weight-semibold">Usage</h5>
+<h5 class="has-text-weight-semibold margin--top--lg">Component Code</h5>
```

Comment:
Instead of adding margin--top--lg to every header element, should we include default top margins for headers in `.sgds-content.has-default-header-styles`? (under _sgds.content.scss)
@rav3 @darenng